pkgs/build-support/build-fhsenv-bubblewrap/default.nix

```
@@ -136,6 +138,7 @@ let
       fi
     done
 
+    # lopp through the entries of /etc in the fhs environment.
```

```suggestion
    # loop through the entries of /etc in the fhs environment.
```
Somehow this got un-fixed after we discussed it previously.

---

done!